go/vt/vttablet/tabletmanager/tm_init.go

```diff
@@ -283,6 +283,7 @@ func (tm *TabletManager) Start(tablet *topodatapb.Tablet, healthCheckInterval ti
 
 	// The following initializations don't need to be done
 	// in any specific order.
+	statsTabletType.Set(topoproto.TabletTypeLString(tm.tmState.tablet.Type))
```

Comment:
Won't it be better to do this inside `exportStats`? Similar to how we are setting keyspace and shard stats.

Reply:
That makes a lot of sense to me. Updated